doc/ref/gappkg.xml

```diff
@@ -27,26 +17,36 @@ integrated into the &GAP; help system.
 All  &GAP;  users   who  develop  new  code  are   invited  to  share
 the  results  of their  efforts  with  other  &GAP; users  by  making
 the  code  and its  documentation  available  in  form of  a  package.
-Information how  to do  this is  available from  the &GAP;  Web pages
+Information how  to do  this is  available from  the &GAP; website
 (<URL>https://www.gap-system.org</URL>)
 and in the &GAP; package <Package>Example</Package>
 (see <URL>https://www.gap-system.org/Packages/example.html</URL>).
 There  are  possibilities  to get  a  package  distributed together  with
```

Comment:
This sentence needs changing.  Perhaps (working from the website): Once a package works and has documentation of its functionality, you should consider how to make it available to other GAP users as a deposited package. Please contact support@gap-system.org so that the GAP Group can make the necessary checks. In these are satisfied, the package can be included in the distribution update mechanism.

Reply:
I'd leave it as it is - there are more details below about depositing a package.